cmd/tether/attach.go

```diff
@@ -51,11 +52,18 @@ type attachServerSSH struct {
 	m sync.Mutex
 
 	// conn is held directly as it's how we stop the attach server
-	conn      *net.Conn
+	conn struct {
+		sync.Mutex
```

Comment:
Yep, will do. It's a litte bit mess right now as we communicate by sharing memory :/